app/models/setting.rb

```diff
@@ -198,10 +208,12 @@ def self.convert_array_to_regexp(array)
 
   def self.create_existing(s, opts)
     bypass_readonly(s) do
-      attrs = column_check([:default, :description, :full_name])
+      encrypted = opts.delete(:encrypted)
+      attrs = column_check([:default, :description, :full_name, :encrypted])
```

Comment:
If you're not mass-assigning :encrypted on line 215 then it perhaps shouldn't be listed here.